### PR TITLE
Iteration 1: Team and NFL

### PR DESCRIPTION
**Pull request by daotoo**
Updated team and NFL for Iteration 1.

Java docs for the methods
 Went kinda fast, might need to double check stuff.

@jcurtiss @seancavanaugh

### CODE REVIEW

File: nflTradeMachine/src/tradeMachine/NFL.java [hunk 1]
```diff
@@ -2,56 +2,140 @@
...
-	
-	public NFL(){
+	ArrayList<Player> players = new ArrayList<Player>();
+
+	/**
+	 * Constructor to create NFL instance
+	 * 
+	 */
+	public NFL() {
 		this.teams = teams;
```
Comment (seancavanaugh) [inline nflTradeMachine/src/tradeMachine/NFL.java:20]: I think you need to have teams and players as params and then set them.

File: nflTradeMachine/src/tradeMachine/NFL.java [hunk 2]
```diff
@@ -2,56 +2,140 @@
...
 
+/**
+ * 
+ * NFL class that acts as the admin. Has control over all teams.
+ * 
+ */
 public class NFL {
-	
+
 	ArrayList<Team> teams = new ArrayList<Team>();
```
Comment (seancavanaugh) [inline nflTradeMachine/src/tradeMachine/NFL.java]: Should make these two fields private

File: nflTradeMachine/src/tradeMachine/NFL.java (hunk 1 above)
Comment (daotoo) [inline nflTradeMachine/src/tradeMachine/NFL.java:20]: Yep, I forgot how to Java. We were just talking about that.

Comment (daotoo) [inline nflTradeMachine/src/tradeMachine/NFL.java:20]: Doing the same for Team as well.

File: nflTradeMachine/src/tradeMachine/Player.java [hunk 3]
```diff
@@ -12,12 +12,12 @@ public Player(){
 		this.name = name;
```
Comment (seancavanaugh) [inline nflTradeMachine/src/tradeMachine/Player.java:12]: Bad constructor

File: nflTradeMachine/src/tradeMachine/Team.java [hunk 4]
```diff
@@ -2,55 +2,159 @@
...
+
+	/**
+	 * Constructor to create a Team instance.
+	 */
+	public Team() {
 		this.players = players;
 		this.salaryCap = salaryCap;
 		this.name = name;
-		this.id = id;		
+		this.id = id;
```
Comment (seancavanaugh) [inline nflTradeMachine/src/tradeMachine/Team.java:25]: Bad constructor

**Comment by daotoo** [comment]
@seancavanaugh Constructors should be updated.

File: nflTradeMachine/src/tradeMachine/Team.java [hunk 5]
```diff
@@ -2,55 +2,159 @@
...
+
+	/**
+	 * Gets the number of players on the team
+	 * 
+	 * @param teamID
+	 *            - the id of the team as a string
+	 * @return the number of players on the team, null if team does not exist.
+	 */
+	public int getNumberOfPlayers(String teamID) {
+		if (teamID == id) {
```
Comment (seancavanaugh) [inline nflTradeMachine/src/tradeMachine/Team.java]: I'm alittle confused on this method, why can't we just return the size?

Comment (daotoo) [inline nflTradeMachine/src/tradeMachine/Team.java]: We could, but thats not what our docs had, and I just left it. I can change it though.

Comment (seancavanaugh) [inline nflTradeMachine/src/tradeMachine/Team.java]: @daotoo Nah, it's fine for now. We can change it in iteration never.

Comment (daotoo) [inline nflTradeMachine/src/tradeMachine/Team.java]: @seancavanaugh Its too late, its done.

**Comment by seancavanaugh** [comment]
+1

**Comment by daotoo** [comment]
Should be good to go for updating the controllers. Havent looked at that yet.

**Comment by seancavanaugh** [comment]
[merge it! merge it! Merge it!](http://youtu.be/yqEeP1acj4Y?t=1m2s)

**Comment by daotoo** [comment]
:joy::joy::joy::joy::joy::joy:

**Pull request merged**